Menu/Menu.js

```diff
@@ -1,10 +1,12 @@
 
-const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
+const toggleMenu = (menu) => {
```

Comment:
Menu could be written with no parentheses since you're only passing one param. (Just throwing it out there.)